Resources/Prototypes/_Goobstation/Entities/Objects/Consumable/Food/ingredients.yml

```
@@ -25,4 +27,74 @@
   - type: Appearance
   - type: SolutionContainerVisuals
     maxFillLevels: 2
-    fillBaseName: packet-solid-
\ No newline at end of file
+    fillBaseName: packet-solid-
+
+- type: entity
+  name: block of freedom cheese
+  parent: FoodBakingBase
+  id: FoodFreedomCheeseBlock
+  description: A solid block of artery-clogging cheese based on an questionable old world recipe. Slice it for your buddies.. or just eat the entire block for yourself, you weirdo. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)